deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -99,6 +107,7 @@ spec:
                   type: string
               type: object
             url:
+              description: URL stores the url of the source cluster. The field is only required for the source cluster object.
```

Comment:
The URL field is needed whenever describing a _remote_ cluster. Whether the cluster is remote/local is not relevant.

Reply:
Updated the desciption.